app-modules/campaign/src/Filament/Resources/CampaignResource/RelationManagers/CampaignActionsRelationManager.php

```diff
@@ -87,6 +87,7 @@ public function table(Table $table): Table
             ])
             ->headerActions([
                 Action::make('create')
+                    ->label('New')
```

Comment:
Let's reference the translation here so that it would be updated with the rest of the default actions.